lib/engine/game/g_1822_pnw/step/track.rb

```diff
@@ -33,10 +33,18 @@ def process_lay_tile(action)
               @round.num_laid_track += 1
               @round.laid_hexes << action.hex
             else
+              forest = @game.forest?(action.hex.tile)
```

Comment:
If you are going to reuse, consider a module

Reply:
There's a total of three lines that are re-used - is that worth making a module for?

Reply:
To me? Yes. Up to you for you.

Reply:
I don't think I will for that.